generators/python-v2/codegen/src/ast/__test__/__snapshots__/TypeInstantiation.test.ts.snap

```diff
@@ -54,3 +54,8 @@ exports[`TypeInstantiation > tuple 1`] = `
 "(1, "two", True)
 "
 `;
+
+exports[`TypeInstantiation > uuid 1`] = `
+"UUID("123e4567-e89b-12d3-a456-426614174000")
```

Comment:
would have expected an import to come through here, is UUID built in now?

Reply:
Imports are currently resolved when a `PythonFile` is written, but not included when a general AstNode is written. This is consistent with previous discussions and existing tests. I'm down to revisit this, but would ask that we do it as a separate effort/PR.